load_dev_env_data.php

```diff
@@ -1,13 +1,22 @@
 <?php
 
-// confirm that we want to clear all data and rebuild the database
-echo "This script will clear all data and rebuild the database\n";
-echo "Are you sure you want to continue? (y/n): ";
-$handle = fopen ("php://stdin","r");
-$line = fgets($handle);
-if(trim($line) != 'y'){
-    echo "ABORTING!\n";
-    exit;
+if ($_ENV["LOAD_DATA"]=="1" || $_ENV["LOAD_DATA"]==1){
```

Comment:
New logic to control the scripts behaviour using environment variables we set from outside as is typical in [twelve-factor](https://12factor.net/config).